edb/schema/utils.py

```diff
@@ -818,30 +817,81 @@ def find_item_suggestions(
     # Never suggest object fragments.
     filters.append(lambda s: not isinstance(s, so.ObjectFragment))
 
-    filtered_suggestions = filter(lambda s: all(f(s) for f in filters),
-                                  suggestions)
+    filtered = filter(lambda s: all(f(s) for f in filters), suggestions)
+
+    # Add display names
+    cur_module_name = modaliases.get(None)
+
+    def get_display_name(suggestion: so.Object) -> str:
+        if isinstance(suggestion, so.QualifiedObject):
+            mod = suggestion.get_name(schema).module
+            if mod == "std" or mod == cur_module_name:
+                return get_nq_name(schema, suggestion)
+
+        return str(suggestion.get_displayname(schema))
+
+    return ((s, get_display_name(s)) for s in filtered)
+
+
+def find_fields_suggestions(
+    schema: s_schema.Schema,
+    item_type: Optional[so.ObjectMeta],
+    compiler_ctx: Optional[ContextLevel],
+) -> Iterable[Tuple[so.Object, str]]:
+    from . import types as s_types
+    from . import objtypes as s_objtypes
+    from edb.ir import typeutils
+
+    if item_type is not s_types.QualifiedType:
```

Comment:
I don't think this `item_type` stuff should be needed?

Reply:
Yup, removed that.